tests/set_code_tx_test.go

```diff
@@ -990,5 +990,5 @@ func TestSetCodeTransaction_IsRejectBeforeAllegro(t *testing.T) {
 	tx := signTransaction(t, chainId, &types.SetCodeTx{}, net.GetSessionSponsor())
 
 	err = client.SendTransaction(context.Background(), tx)
-	require.ErrorContains(t, err, "transaction type not supported")
+	require.ErrorContains(t, err, "empty authorization")
```

Comment:
This defeats the original purpose of this test. You should add an authorization to the transaction to make it pass this static check to actually get to the point where the transaction type is checked.

Reply:
https://github.com/0xsoniclabs/sonic/pull/200/commits/730cc402e58a98f26bdde28704fbde8a849ccae9